jupyter_server_proxy/handlers.py

```diff
@@ -28,14 +28,17 @@ def get(self, *args):
         dest = src._replace(path=src.path + '/')
         self.redirect(urlunparse(dest))
 
-class LocalProxyHandler(WebSocketHandlerMixin, IPythonHandler):
-
+class ProxyHandler(WebSocketHandlerMixin, IPythonHandler):
```

Comment:
Can you make a note here about how this should be only used by classes that are overriding this from someplace else, and not directly, to prevent bypassing CORS?

Reply:
Sure.